src/apps/phuture/phuture.definition.ts

```diff
@@ -0,0 +1,50 @@
+import { Register } from '~app-toolkit/decorators';
+import { appDefinition, AppDefinition } from '~app/app.definition';
+import { AddressFormat, AppAction, AppTag, GroupType } from '~app/app.interface';
+import { Network } from '~types/network.interface';
+
+export const PHUTURE_DEFINITION = appDefinition({
+  id: 'phuture',
+  name: 'Phuture',
+  tags: [AppTag.ASSET_MANAGEMENT, AppTag.FUND_MANAGER, AppTag.STAKING],
+  keywords: [], // TODO: add keywords
```

Comment:
@olivermehr

Reply:
Would remove the staking tag. 

keywords: index funds, asset management